config/manifests/bases/manila-operator.clusterserviceversion.yaml

```diff
@@ -30,33 +30,29 @@ spec:
       kind: ManilaShare
       name: manilashares.manila.openstack.org
       version: v1beta1
-  description: manila
-  displayName: manila
-  icon:
-  - base64data: ""
-    mediatype: ""
+  description: Manila Operator
+  displayName: Manila Operator
   install:
     spec:
       deployments: null
     strategy: ""
   installModes:
-  - supported: false
+  - supported: true
     type: OwnNamespace
-  - supported: false
+  - supported: true
     type: SingleNamespace
   - supported: false
     type: MultiNamespace
   - supported: true
     type: AllNamespaces
   keywords:
-  - manila
+  - openstack
```

Comment:
(this is for follow-up) I understand this is consistent with cinder and glance but the other oeprators are adding different items (for example keystone operator has OpenStack, Identity and Keystone).
We probably have to update cinder/glance/manila operator to follow that standard later.

Reply:
Sure, let's fix the main issue here and follow up on that minor thing, feel free to propose a patch to align the operators (I'll review!)